FirebaseAuth/Sources/Swift/User/User.swift

```diff
@@ -2082,7 +2088,9 @@ extension User: NSSecureCoding {}
   private let kMultiFactorCodingKey = "multiFactor"
   private let kTenantIDCodingKey = "tenantID"
 
-  public static var supportsSecureCoding = true
+  public static var supportsSecureCoding: Bool {
+    return true
+  }
```

Comment:
should this be internal too?
```suggestion
  static var supportsSecureCoding: Bool = true
```